src/main/services/download/torbox-downloader.ts

```diff
@@ -0,0 +1,151 @@
+import { Game } from "@main/entity";
+import { TorBoxClient } from "../torbox";
+import { HttpDownload } from "./http-download";
+import { GenericHttpDownloader } from "./generic-http-downloader";
+import { gameRepository } from "@main/repository";
+// import { calculateETA } from "./helpers";
```

Comment:
No need for calculating ETA. Left it here in case in the future TorBox gives a definite file size for zips.